assets/js/modules/search-console/datastore/service.js

```diff
@@ -53,6 +55,19 @@ export const selectors = {
 		}
 		return addQueryArgs( baseURI, queryArgs );
 	} ),
+
+	/**
+	 * Checks whether the property is a Search Console Domain Property.
```

Comment:
Maybe slightly more accurate would be:
```suggestion
	 * Checks whether the Search Console property is a domain property.
```
(it _always_ is a Search Console property)